src/components/GitmojiList/index.js

```diff
@@ -16,7 +17,8 @@ type Props = {
 }
 
 const GitmojiList = (props: Props): Element<'div'> => {
-  const [searchInput, setSearchInput] = React.useState(null)
+  const { query } = useRouter()
+  const [searchInput, setSearchInput] = React.useState('')
```

Comment:
I think we can easily replace the `useEffect` with this:

```suggestion
  const [searchInput, setSearchInput] = React.useState(router.query.search || '')
```

Reply:
Unfortantely, Next doesn't populate the `router.query` object, so we have to use an effect that listens for when Next eventually populates the `router.query` object.